bindings/gumjs/gumdukinterceptor.h

```diff
@@ -27,6 +27,11 @@ struct _GumDukInterceptor
   GumDukHeapPtr invocation_context;
   GumDukHeapPtr invocation_args;
   GumDukHeapPtr invocation_retval;
+
+  GumDukHeapPtr cached_invocation_args;
+  gboolean cached_invocation_args_in_use;
+  GumDukHeapPtr cached_invocation_context;
+  gboolean cached_invocation_context_in_use;
```

Comment:
Let's put the two `cached_invocation_context*` variables first so they follow the same order as they're used.